src/stdlib_ascii.fypp

```diff
@@ -17,6 +20,17 @@ module stdlib_ascii
 
     ! Character conversion functions
     public :: to_lower, to_upper, to_title, reverse
+    public :: to_char
+
+    !> Version: experimental
+    !>
+    !> Create a character string representing the value of the provided variable.
+    interface to_char
+    #:for kind in INT_KINDS
+        module procedure :: integer_${kind}$_to_char
+        module procedure :: logical_${kind}$_to_char
```

Comment:
Here, and everywhere else in the module, I suggest flipping this around so that the base name of the function comes first, type second, kind third:

```
        module procedure :: to_char_integer_${kind}$
￼        module procedure :: to_char_logical_${kind}$
```

This is consistent with other templates in stdlib, and the left-to-right reading order in most languages.